integration_tests/dbt_project.yml

```diff
@@ -20,8 +20,7 @@ vars:
     recharge_customer_identifier: "customer_data"
     recharge_discount_identifier: "discount_data"
     recharge_one_time_product_identifier: "one_time_product_data"
-    order: "{{ ref('order_data') }}" # Comment out for generating docs -- Uncomment this before sending to buildkite.
-    # recharge_order_identifier: "order_data" # Uncomment for generating docs -- Comment this out before sending to buildkite.
+    recharge_orders_identifier: "orders_data"
```

Comment:
What are your thoughts on still including `order` and then adjusting the `run_models.sh` to test when the `recharge__using_orders` variable is true or false to ensure the code still works with pre and post June 18th connectors?

Reply:
I agree this is the better route, thanks! I have updated.